package.json

```diff
@@ -58,25 +58,25 @@
     "stylus": "^0.51.1"
   },
   "devDependencies": {
-    "babel-eslint": "^3.1.23",
+    "babel-eslint": "^4.1.3",
     "chalk": "^1.1.0",
     "glob-manipulate": "^1.1.1",
     "gulp": "^3.9.0",
     "gulp-babel": "^5.1.0",
-    "gulp-eslint": "^0.15.0",
-    "gulp-filter": "^2.0.2",
+    "gulp-eslint": "^1.0.0",
+    "gulp-filter": "^3.0.1",
     "gulp-load-plugins": "^1.0.0-rc.1",
     "gulp-plumber": "^1.0.1",
     "gulp-rename": "^1.2.2",
     "gulp-src-ordered-globs": "^1.0.3",
     "gulp-util": "^3.0.6",
     "gulp-watch": "^4.3.1",
-    "merge-stream": "^0.1.8",
+    "merge-stream": "^1.0.0",
     "mocha": "^2.2.5",
     "should": "^7.0.2",
     "stream-array": "^1.1.0",
     "through2": "^2.0.0",
-    "vinyl-paths": "^1.0.0"
+    "vinyl-paths": "^2.0.0"
```

Comment:
vinyl-paths@2 changed the API to use promises, we'd need to update its usage in the gulpfile/es20xx.